lib/types/webrtc.d.ts

```diff
@@ -6,6 +6,7 @@ export type RTCPeerConfig = {
     simulcast?: boolean;
     connTimeoutMs?: number;
     dcSignaling?: boolean;
+    enableAV1?: boolean;
```

Comment:
We need `RTCPeer` to be aware of the global setting now since we'll be handling codecs entirely on this side from now on.